### PR TITLE
Added Spanish Readme

### PR DESCRIPTION
**Pull request by bryan-trz**
### All Submissions:

- [ :white_check_mark:] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/reflex-dev/reflex/blob/main/CONTRIBUTING.md) file?
- [:white_check_mark: ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/reflex-dev/reflex/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ :white_check_mark:] This change requires a documentation update

### Description
Hi! I have added a Spanish translation of the Readme file. 
Will continue fixing typos. Just wanted to translate the main parts of the readme first!

### CODE REVIEW

File: docs/es/README.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    """The app state."""
+    prompt = ""
+    image_url = ""
+    processing = False
+    complete = False
+```
+
+El estado defines all the variables (called vars) in an app that can change and the functions that change them.
+
+Here the state is comprised of a `prompt` and `image_url`. There are also the booleans `processing` and `complete` to indicate when to show the circular progress and image.
```
Comment (picklelo) [inline docs/es/README.md:161]: And this section?

Comment (bryan-trz) [reply docs/es/README.md]: Just fixed that!

**Review by picklelo — APPROVED** [review]
thanks for adding!

**Comment by cllatser** [comment]
Good morning, you have done a good job, but I have seen that there are spelling and grammatical mistakes. At least in the Spanish of Spain. What country are you from? How can I correct the mistakes or do a PR of Spanish (Spain). Greetings :)

**Comment by picklelo** [comment]
@cllatser Feel free to make a new PR to fix anything!

**Pull request merged**